metadata.rb

```diff
@@ -7,5 +7,4 @@
 
 source_url 'https://github.com/chef-cookbooks/chef-ingredient'
 issues_url 'https://github.com/chef-cookbooks/chef-ingredient/issues'
-
-chef_version '>= 12.5'
+chef_version '>= 12.5' if respond_to?(:chef_version)
```

Comment:
We need this so old clients will properly fail. In < 12.6 with the metadata they freak out on unknown values. This makes sure we fail right on 12.0 - 12.5